test/optimizer/column_stats_test.cpp

```diff
@@ -0,0 +1,39 @@
+#include "optimizer/statistics/column_stats.h"
+#include "gtest/gtest.h"
+
+#include "util/test_harness.h"
+
+namespace terrier::optimizer {
+// NOLINTNEXTLINE
+TEST(ColumnStatsTests, GetColumnIDTest) {
+  auto column_stats_obj = ColumnStats(catalog::db_oid_t(1), catalog::table_oid_t(1), catalog::col_oid_t(1), 10, 4, 0.2,
```

Comment:
nit - You could refactor this to create the object once in the test class and then just reuse it in all of the individual test cases.